ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/MockExecutorService.java

```diff
@@ -55,6 +56,10 @@ public void runPendingFuturesInSeparateThreads(final ExecutorService executorSer
     currentTasks.forEach(task -> executorService.execute(task::run));
   }
 
+  public List<ExecutorTask<?>> getTasks() {
```

Comment:
I don't think you need to expose this?  You can just use the existing `getPendingFuturesCount()` or `getFutures()`.

Reply:
Done.